gpflow/models/__init__.py

```diff
@@ -18,6 +18,11 @@
 from .gpmc import GPMC
 from .gpr import GPR
 from .model import BayesianModel, GPModel
+from .mixins import (
```

Comment:
```suggestion
from .training_interface import (
```

Reply:
(as it defines all the types I think "mixin" is rather confusing; it does all relate to 'data' so "training_interface" seems appropriate)

Reply:
i think mixins keeps it clearer that the pattern we are going for is a mixin pattern